roles/installer/files/filetranspile-1.1.1.py

```
@@ -119,7 +118,7 @@ def merge(self):
         # Walk through the files and append them for merging
         all_files = []
         all_links = []
-        for root, _, files in os.walk(self.fake_root):
+        for root, unused_dirs, files in os.walk(self.fake_root):
```

Ruff (0.15.18)

[warning] Loop control variable `unused_dirs` not used within loop body

Rename unused `unused_dirs` to `_unused_dirs`

(B007)